code/modules/research/designs.dm

```diff
@@ -463,7 +463,7 @@
 	..()
 	name = "Implantable biocircuit design ([item_name])"
 
-/datum/design/item/implant/chemical
+/* /datum/design/item/implant/chemical
```

Comment:
Leave the implants in for now. This is a discussion I have yet to have with West, who explicitly wanted us to review them.